### PR TITLE
add dev.salaniLeo.forecast

### PR DESCRIPTION
**Pull request by SalaniLeo**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

File: dev.salaniLeo.forecast.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id": "dev.salaniLeo.forecast",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
```
Comment (hfiguiere) [inline dev.salaniLeo.forecast.json:4]:
```suggestion
    "runtime-version": "44",
```

File: dev.salaniLeo.forecast.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/SalaniLeo/Forecast",
+                    "tag" : "0.1.0",
+                    "commit" : "e64f86b2a18eb2f0daa9b0a18d3c952fc5b29fd3"
+                }
+            ],
+            "dependencies": [
```
Comment (hfiguiere) [inline dev.salaniLeo.forecast.json:36]: This is invalid:
```
(flatpak-builder:2): flatpak-builder-WARNING **: 18:10:04.340: Unknown property dependencies for type BuilderModule
```

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
everything works, ready to merge

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
the missing icons should now be there

**Comment by SalaniLeo** [comment]
bot, build dev.salaniLeo.forecast

**Comment by SalaniLeo** [comment]
can merge

**Pull request closed without merging**